### ISSUE TITLE
Linux package/Building from source question

### ISSUE THREAD

**Issue by micwoj92** [Manabender/Cursed-Blocks#1]
There are no Linux releases and I really want to play this game (and avoid using wine). I could try compiling from source but I have 0 idea how to work with C# projects. Any help appreciated, TIA.

**Comment by Manabender** [comment]
I just uploaded a Linux build for the latest version. I unfortunately have no way to test it myself at the moment. Please let me know how it goes!

**Comment by micwoj92** [comment]
Thanks, it works fine. Still interested in build instructions.

**Comment by Manabender** [comment]
The game is made in Unity, specifically version 2019.4.20f1. In theory, if you put all the source files in a folder and open it as a Unity project in the Unity editor, you should be able to build an executable from there. That being said, I would be very surprised if it was actually that simple.

Unfortunately, my experience with Linux is _extremely_ limited, so I'm afraid I'm unable to be of much assistance.

**Comment by micwoj92** [comment]
I did it. Downloaded unity editor (I used 2019.4.25f1 lts version), activated personal free license, then cloned the project, opened in unity, it gave some message about non matching editor version, so I just clicked on Continue, it updated some files and after it finished it gave warnings about png files missing, so then I went to GitHub, noticed "Stored with Git LFS" label, so I did the standard `git lfs install` and `git lfs pull` to download these pngs, opened in unity again and I could build and it worked fine. Thanks for assistance.
